src/componentsTest/Forums.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useState, useEffect } from "react"
```

Comment:
ideally, you would comment out the portions that are breaking the file instead of deleting them, making it easier to see what was removed. But, I'm over all the unit test stuff so if it works it works 🥔

Reply:
@gabbytang thanks for adding them back :)